src/json-patch-ot-agent.js

```diff
@@ -73,3 +73,14 @@ JSONPatchOTAgent.applyOT = function(callback){
         callback(obj, consecutivePatch);
 	};
 };
+
+/**
+ * Reset queue internals and object to new, given state
+ * @param obj object to apply new state to
+ * @param newState versioned object representing desired state along with versions
+ */
+JSONPatchOTAgent.prototype.reset = function(obj, newState){
+	this.ackLocalVersion = newState[this.localPath.replace(/^\//, '')];
```

Comment:
I suppose functions for accessing properties by json path is in json-patch? If so, I propose just leaving it here and writing up a bug, it's unlikely someone will use a complex path, right?